pkg/provisioner/ironic/ironic.go

```diff
@@ -822,6 +822,20 @@ func (p *ironicProvisioner) getUpdateOptsForNode(ironicNode *nodes.Node, checksu
 	return updates, nil
 }
 
+func (p *ironicProvisioner) startManualCleaning(ironicNode *nodes.Node) (result provisioner.Result, err error) {
+	// Set raid config
+	err = p.setTargetRAIDCfg(ironicNode)
+	if nil != err {
+		return
+	}
+	// Build clean steps
+	cleanSteps := p.buildProvisionCleanSteps()
+	// Start manual clean
+	p.log.Info("start manual cleaning with", "steps", cleanSteps)
```

Comment:
I don't think we want to talk about "manual" cleaning. In the metal3 context, all of this is automated, right? Is someone not familiar with Ironic going to understand what "cleaning" is at all? Could we say "removing existing configuration from host" or even "preparing RAID configuration"?

Reply:
@dhellmann I would actually really like control over the cleaning configuration.  For my setup, we do not want the non-root volumes to be destroyed automatically so we can preserve data.  In AWS, you have a choice about terminating your non-root (additional EBS) volumes, so this would provide similar functionality.  

One use case is we have logs that are written there and may not have been shipped out due to whatever failure tainted the node.  We have an operator that will find all the orphan EBS volumes, ship logs it finds, then terminate it.  

Another case is we are using some local persistent volumes on-prem, and need to be able to persist that data beyond provisions